Project-Timer-frontend/src/app/dashboard/projects/projects.component.html

```diff
@@ -0,0 +1,7 @@
+<ol>
```

Comment:
po co to <ol>  ?

Reply:
żeby była numeracja projektów jak się wezmę za stylizację to wywalę.